L1Trigger/TrackFindingTracklet/interface/TrackletEventProcessor.h

```diff
@@ -63,6 +68,8 @@ namespace trklet {
     Timer PDTimer_;
 
     std::vector<Track> tracks_;
```

Comment:
EDProducer data members, such as streamsTrack_ are not thread safe, especially if their value changes every event. So this makes it harder in future for us to make this EDProducer multithreaded. Though data member tracks_ already had the same problem ...

Reply:
TrackletEventProcessor is not an EDProducer. My code looks thread safe to me.

Reply:
"TrackletEventProcessor eventProcessor" is a data member of the EDProducer L1TrackFPGAProducer. And tracks_ is a data member of TrackletEventProcessor, and therefore indirectly also of the L1TrackFPGAProducer.

Reply:
ah who would have thought that this one is data member, shall we construct it in produce?

Reply:
done